### PR TITLE
Announced messages while drunk will be slurred

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
tweak: Announcing messages while drunk will be slurred
/:cl:

### CODE REVIEW

**Comment by Potato-Masher** [comment]
Now we just need a way for the AI to get drunk.

**Comment by tortellinitony** [comment]
porting more shit from hippie to tg I see?

**Comment by davethwave** [comment]
viruses?

**Comment by MoreRobustThanYou** [comment]
BSAing an AI makes it stutter.

**Comment by MoreRobustThanYou** [comment]
BSAing an AI makes it stutter.

**Comment by MoreRobustThanYou** [comment]
BSAing an AI makes it stutter.

**Comment by coiax** [comment]
Wait, are announcements actually verbal announcements over some sort of PA system? Do you still hear announcements if you're deaf?

**Comment by MoreRobustThanYou** [comment]
@coiax https://github.com/tgstation/tgstation/blob/b2b3f7f7c0cbb357a5452bc6f31e3f8ee2b7d9ab/code/__HELPERS/priority_announce.dm#L34

File: code/controllers/subsystem/communications.dm [hunk 1]
```diff
@@ -23,7 +23,14 @@ SUBSYSTEM_DEF(communications)
 		minor_announce(html_decode(input),"[user.name] Announces:")
 		silicon_message_cooldown = world.time + COMMUNICATION_COOLDOWN_AI
 	else
-		priority_announce(html_decode(input), null, 'sound/misc/announce.ogg', "Captain")
+		input = html_decode(input)
```
Comment (ninjanomnom) [inline code/controllers/subsystem/communications.dm]: you don't need to html_decode before any of these procs

**Review by ninjanomnom — DISMISSED** [review]

**Comment by coiax** [comment]
But they're still in a universal language format. Someone needs to fix that.

**Comment by Bawhoppen** [comment]
@coiax It was unclear for a long time whether they counted as verbal audio or some kind of text-to-speech system, and nobody dared to try and set the precedent for what they would be considered.
This PR will be pivotal in deciding it, so I hope maintainers make the right choice.

Of course, if we choose the path of it being just an intercom, then we need to make sure mimes can't use it, fix the language issues, etc etc.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Clean the dupe code

File: code/game/machinery/requests_console.dm [hunk 2]
```diff
@@ -302,6 +302,9 @@ GLOBAL_LIST_EMPTY(allConsoles)
 	if(href_list["sendAnnouncement"])
 		if(!announcementConsole)
 			return
+		if(isliving(usr))
+			var/mob/living/L = usr
+			message = L.treat_message(input)
```
Comment (ShizCalev) [inline code/game/machinery/requests_console.dm]: message = L.treat_message(message)

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**